site/layouts/shortcodes/alpha.html

```diff
@@ -1,9 +1,19 @@
-{{- $gate := .Get "gate" }}
+{{- $gate := split (.Get "gate") "," }}
+{{- $len_gate := len $gate }}
 {{- $title := .Get "title" }}
 <div class="alert alert-warning" role="alert">
     <h4 class="alert-heading">Warning</h4>
-    <p>The {{ $title }} feature is currently <strong><a href="{{ ref . "/docs/Guides/feature-stages.md#alpha" }}">Alpha</a></strong>,
+    <p>The {{ $title }} {{ if gt $len_gate 1 }}features are{{ else }}feature is{{ end }} currently <strong><a href="{{ ref . "/docs/Guides/feature-stages.md#alpha" }}">Alpha</a></strong>,
         not enabled by default, and may change in the future.</p>
-    <p>Use the Feature Gate <code>{{- $gate }}</code> to enable and test this feature.</p>
+    <p>Use the Feature
+        {{ if gt $len_gate 1 }}Gates{{else}}Gate{{ end }}
+        {{- range $index, $value := first (sub $len_gate 1) $gate }}
+        {{ if ne $index 0 }},{{ end }} <code>{{ $value }}</code>
+        {{- end}}
+        {{ if gt $len_gate 1 }}and{{ end }}
```

Comment:
nit: I'd personally change this to `, and` to use an oxford comma, but I'm also OK if you prefer to not use the oxford comma.

Reply:
The eternal debate. Added!